### ISSUE TITLE
Home page

### ISSUE THREAD

**Issue by AlbaPi** [TheRosettaFoundation/SOLAS-Match#1096]
“Home page” on Trommons is not working properly. It doesn’t appear it all the tasks that it should be there. For example, when I choose Spanish>English, it just appear one task and there are currently 5 tasks available. 

Furthermore, when I log in, the first tasks that appear it in my home page are Latvian tasks, and I don’t speak Latvian.

**Comment by aquascene** [comment]
This is probably due to the Backend being down.  As far as I know the matching algorithm is controlled by the Backend.  Looking into this.

**Comment by aquascene** [comment]
This issue may now be resolved as the Backend is up and running again.  Please check and let me know if it is still an issue.

**Comment by AlbaPi** [comment]
I'm still seeing some Latvian tasks in the second position...

**Comment by Wasala** [comment]
@AlbaPi Is this still happening even after you've set your native language?

**Comment by AlbaPi** [comment]
Yes, I am still seeing some Latvian tasks. Then I am seeing English, Spanish tasks. So maybe there's somet
